go/ql/src/experimental/frameworks/DecompressionBombs.qll

```diff
@@ -676,45 +713,60 @@ module DecompressionBombs {
           this = f.getACall().getArgument(0)
         )
         or
-        exists(Method f |
-          f.hasQualifiedName("bufio", "Reader",
-            ["Read", "ReadBytes", "ReadByte", "ReadLine", "ReadRune", "ReadSlice", "ReadString"])
+        exists(Method m |
+          m.hasQualifiedName("bufio", "Reader",
+            ["ReadBytes", "ReadByte", "ReadLine", "ReadRune", "ReadSlice", "ReadString"])
+        |
+          this = m.getACall().getReceiver()
```

Comment:
Some sink methods, such as these ones, don't have `hasFlowToComparison` applied to the return value. Is that intentional?